src/app/debug/table/table.component.ts

```diff
@@ -569,7 +576,7 @@ export class TableComponent implements OnInit, OnDestroy {
 
   calculateViewDropDownWidth(): void {
     let longestViewName = '';
-    for (let [key, value] of Object.entries(this.viewSettings.views)) {
+    for (let [key] of Object.entries(this.viewSettings.views)) {
```

Comment:
Wouldn't this work as well?
```suggestion
    for (let key of this.viewSettings.views.keys()) {
```
Otherwise the following: 
```suggestion
    for (let key of Object.keys(this.viewSettings.views)) {
```

Reply:
went for the second option. first option did not work